### PR TITLE
Fix the compatibility issue with loading WATM-Go in WATER-rs

### PR DESCRIPTION
**Pull request by erikchi**
Solve this issue: https://github.com/refraction-networking/watm/issues/2

### CODE REVIEW

File: crates/water/src/runtime/core.rs [hunk 1]
```diff
@@ -37,10 +38,13 @@ impl H2O<Host> {
...
         let engine = Engine::new(&wasm_config)?;
-        let linker: Linker<Host> = Linker::new(&engine);
-
+        
         let module = Module::from_file(&engine, &conf.filepath)?;
+        
+        let linker: Linker<Host> = Linker::new(&engine);
 
+        // linker.allow_unknown_exports(true);
+
```
Comment (gaukas) [inline crates/water/src/runtime/core.rs:47]: Are these just pure formatting or is the ordering making any differences?

**Review by gaukas — APPROVED** [review]
Thanks for the change, looks good to me!

File: crates/water/src/runtime/core.rs (hunk 1 above)
Comment (gaukas) [reply crates/water/src/runtime/core.rs:47]: If the ordering matters, it is better to leave some explicit comments explaining so. If just styling/formatting, I guess it is fine to leave it like that.

**Comment by gaukas** [comment]
Seems code format check doesn's like it 😅

Is there a way to automate the formatting update on saving a .rs file? (Like how VS Code works with gofmt on save).

File: crates/water/src/runtime/core.rs (hunk 1 above)
Comment (erikziyunchi) [reply crates/water/src/runtime/core.rs]: The order doesn't matter, it's just styling :D.

**Comment by erikziyunchi** [comment]
> Is there a way to automate the formatting update on saving a .rs file? (Like how VS Code works with gofmt on save).

Good point, I should search this out. It would be nice to have that feature similar to what Go has.

**Pull request merged**